x-pack/legacy/plugins/epm/common/types.ts

```diff
@@ -30,34 +31,49 @@ export enum ElasticsearchAssetType {
   ilmPolicy = 'ilm-policy',
 }
 
-// Registry's response types
-// from /search
-// https://github.com/elastic/package-registry/blob/master/docs/api/search.json
-export type RegistryList = RegistryListItem[];
-export interface RegistryListItem {
-  description: string;
-  download: string;
-  icon: string;
+// from /package/{name}
+// type Package struct at https://github.com/elastic/package-registry/blob/master/util/package.go
+// https://github.com/elastic/package-registry/blob/master/docs/api/package.json
+export interface RegistryPackage {
   name: string;
-  version: string;
-  title?: string;
-}
-
-export interface ScreenshotItem {
-  src: string;
   title?: string;
+  version: string;
+  readme?: string;
+  description: string;
+  type: string;
+  categories: string[];
+  requirement: RequirementsByServiceName;
+  screenshots?: ScreenshotItem[];
+  icons?: string[];
+  assets?: string[];
+  internal?: boolean;
+  format_version: string;
+  datasets?: Dataset[];
+  download: string;
+  path: string;
 }
 
-export type ServiceName = 'kibana' | 'elasticsearch';
-
-// from /package/{name}
-// https://github.com/elastic/package-registry/blob/master/docs/api/package.json
 export type RequirementVersion = string;
 export type RequirementVersionRange = string;
 export interface ServiceRequirements {
   versions: RequirementVersionRange;
 }
 
+// Registry's response types
+// from /search
+// https://github.com/elastic/package-registry/blob/master/docs/api/search.json
+export type RegistryList = RegistryListItem[];
+// from getPackageOutput at https://github.com/elastic/package-registry/blob/master/search.go
+export type RegistryListItem = Pick<
```

Comment:
Build `RegistryListItem` from `RegistryPackage` which is also how EPR does is

Reply:
Should we call this perhaps `RegistrySearchItem` to make it more clear it comes from the search endpoint?